### PR TITLE
Added CCSDSPy to package index

### PR DESCRIPTION
**Pull request by ehsteve**
Sorry, i accidentally created a branch for this which I did not mean to do! Should have done it from my fork.

### CODE REVIEW

File: _data/projects.yml [hunk 1]
```diff
@@ -35,6 +35,14 @@
...
   keywords: ["magnetosphere","ionosphere_thermosphere_mesosphere","specific"]
 
+- name: CCSDSPy
+  code: https://github.com/ccsdspy/ccsdspy/
+  description: A Python package for reading CCSDS (spacecraft) packet data.
+  docs: "https://docs.ccsdspy.org/en/latest/#"
+  logo: https://docs.ccsdspy.org/en/latest/_static/logo.png
+  contact: Daniel da Silva
+  keywords: ["ccsds","specific"]
+
```
Comment (sapols) [inline _data/projects.yml:45]: This is missing the grading badges required to add a new project.

**Review by sapols — CHANGES_REQUESTED** [review]

**Comment by sapols** [comment]
Thanks for the PR Steve. We're happy to add CCSDSPy to PyHC, but I can't do so until the grading badges are added.

**Comment by ddasilva** [comment]
> Thanks for the PR Steve. We're happy to add CCSDSPy to PyHC, but I can't do so until the grading badges are added.

Is this something I need to add to CCSDPy or is it something that needs to be added to this merge request? I'm not sure what a grading badge is!

**Comment by sapols** [comment]
Steve would know, I'm pretty sure he designed the badges. Haha

It's something you add to the PR; a self-evaluation of how well the package adheres to PyHC's [standards](https://github.com/heliophysicsPy/standards/blob/main/standards.md) (the colorful badges you see across our [Projects page](https://heliopython.org/projects/)). See our [project addition instructions](https://heliopython.org/docs/adding_a_project/) to read more.

File: _data/projects.yml (hunk 1 above)
Comment (ehsteve) [reply _data/projects.yml:51]: Added the badges.

**Comment by ddasilva** [comment]
> @ddasilva I assessed everything as green. Hopefully you agree. If so the merge request should be ready to be reviewed again.

Looks good to me!

File: _data/projects.yml [hunk 2]
```diff
@@ -35,6 +35,20 @@
   contact: Bryan Harter, Michael Liu, David Stansby, Michael Hirsch
   keywords: ["magnetosphere","ionosphere_thermosphere_mesosphere","specific"]
 
+- name: CCSDSPy
+  code: https://github.com/ccsdspy/ccsdspy/
+  description: A Python package for reading CCSDS (spacecraft) packet data.
+  docs: "https://docs.ccsdspy.org/en/latest/#"
+  logo: https://docs.ccsdspy.org/en/latest/_static/logo.png
+  contact: Daniel da Silva
+  keywords: ["ccsds","specific", "heliosphere", "ionosphere_thermosphere_mesosphere", "magnetosphere"]
```
Comment (sapols) [inline _data/projects.yml:44]: Looks good! "ccsds" wasn't a valid keyword from the taxonomy but I added it so it is now 👍

**Review by sapols — APPROVED** [review]

**Comment by sapols** [comment]
Welcome to PyHC, CCSDSPy!

**Comment by ddasilva** [comment]
Thank you!

**Pull request merged**